README.md

```diff
@@ -5,13 +5,24 @@
   >
 </h3>
 
+<div align="center">
+
+  <a href="">![https://pypi.python.org/pypi/unstructured/](https://img.shields.io/pypi/l/unstructured.svg)</a>
+  <a href="">![https://pypi.python.org/pypi/unstructured/](https://img.shields.io/pypi/pyversions/unstructured.svg)</a>
+  <a href="">![https://GitHub.com/unstructured-io/unstructured.js/graphs/contributors](https://img.shields.io/github/contributors/unstructured-io/unstructured)</a>
+  <a href="">![code_of_conduct.md](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg) </a>
```

Comment:
code_of_conduct.md does not exist in this repo yet so broken link

Reply:
ah, nvm, see all links are not filled in yet.

Reply:
thanks, just recently added it

Reply:
[CODE_OF_CONDUCT.md](https://github.com/Unstructured-IO/unstructured/blob/main/CODE_OF_CONDUCT.md)

Reply:
oh I will fix the links then thanks

Reply:
DONE

Reply:
shall we count this as done? @MthwRobinson @cragwolfe

Reply:
Looks good to me!